_drafts/2017-06-07-checkers.md

```diff
@@ -0,0 +1,115 @@
+---
+name: checkers
+layout: post_discourse
+title: Tackling the Research Compendium at #runconf17
+authors:
+  - name: Noam Ross
+    url: http://twitter.com/noamross
+  - name: Alice Daish
+    url: http://twitter.com/alice_data
+  - name: Laura DeCicco
+    url: http://twitter.com/DeCiccoDonk 
+  - name: Nistara Randhawa
+    url: http://twitter.com/nistara_r
+  - name: Jennifer Thompson
+    url: http://twitter.com/jent103
+  - name: Nick Tierney
+    url: https://twitter.com/nj_tierney
+categories:
+  - blog
+tags:
+  - unconf
+  - testing
+  - review
+  - packages
+  - software
+  - community
+  - meetings
+---
+
+Two years ago at [\#runconf15], there was a great discussion about best practices for organizing R-based analysis projects that yielded a [nice guidance document] describing *research compendia*. Compendia, as we described them, were minimal products of reproducible research, using parts of R package structure to organize the inputs, analyses, and outputs of research projects.
+ 
+Since then, we've seen more examples and models of research compendia emerge (the organization of such projects is [something of an obsession] for some of the community). In parallel, there's been much progress on a number of fronts with R *packages*: ROpenSci's [package review process] has expanded and we've worked out many kinks. Infrastructure for automated testing of package code has been developed and field tested. So at [\#runconf17](unconf17.ropensci.org), we wanted to see how much of this progress in review, testing, and automation could apply to research compendia.
```

Comment:
@stefaniebutland I told Noam that we do rOpenSci. That's how we've always written it.